wapi/curves.py

```diff
@@ -320,7 +320,8 @@ def get_data(self, tag=None, data_from=None, data_to=None, time_zone=None, filte
 
 class InstanceCurve(BaseCurve):
     def search_instances(self, issue_date_from=None, issue_date_to=None,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method "search_instances" has 18 parameters, which is greater than the 15 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dev.wattsight.com/coding_rules#rule_key=python%3AS107)